RadixWallet/Features/DappInteractionFeature/Children/AccountPermission/AccountPermission+View.swift

```diff
@@ -44,7 +44,7 @@ extension AccountPermission {
 			) { viewStore in
 				GeometryReader { geometry in
 					ScrollView {
-						VStack(spacing: .medium2) {
+						VStack(spacing: .large1) {
```

Comment:
Not sure if this value is correct since Zeplin is indicating a padding to a wrong part of the view
![Screenshot 2024-07-17 at 16 58 17](https://github.com/user-attachments/assets/0faf9471-19dd-4ab0-b67a-7e619a7c224a)

Reply:
but are we sure that 69 represents 3 lines? and not just a random box that happens to look like it represents 3 lines from the Text we need to calculate the distance?

I believe we should push for having updated and consistent designs before making these changes. Not only it would be easier for us, but also state the source of truth if implementation differs among platforms.

Reply:
Fair enough, asked Aftab